### DISCUSSION TITLE
add some scripts in pyaedt's code that uses ironpython

### DISCUSSION THREAD

**Discussion by aknsj** [ansys/pyaedt#4419] [Q&A]
Hi all. If I want to add some scripts in pyaedt's code that uses ironpython i.e. ScriptEnv library, how do I need to do it?
Looking forward for your answer.

**Comment by gmalinve** [comment]
Hi @aknsj,

Could you explain a bit better what you are trying to do?
Why do you need ironpython within pyaedt?

Thanks

Giulia
